patcher/src/ts/redux/modules/servers.ts

```diff
@@ -158,11 +158,17 @@ export default function reducer(state: ServersState = initialState, action: any
         isFetching: true
       });
     case FETCH_SERVERS_SUCCESS:
+      let selectedServer:Server;
```

Comment:
https://github.com/airbnb/javascript#blocks--braces

    if (expression) statement;

or

    if (expression) {
      statement;
    }

are the acceptable forms.